dfc/tests/regression_test.go

```diff
@@ -88,7 +93,115 @@ var (
 	failLRU = ""
 )
 
-func Test_regression(t *testing.T) {
+// Intended for a deployment with multiple targets
+// 1. Unregister target T
+// 2. Create local bucket
+// 3. PUT large amount of objects into the local bucket
+// 4. GET the objects while simultaneously re-registering the target T
+func TestGetAndReRegisterInParallel(t *testing.T) {
+	var (
+		sid                string
+		num                = 20000
+		numGetsForEachFile = 5
+		numGetErrs         int
+		maxNumGetErrs      = num * numGetsForEachFile / 10000 // 0.01% of GET requests
```

Comment:
0.01% is not 10,000, this is 10%
add an explanation of why less than this % is expected to fail.

Reply:
Yup, forgot to change the percentage in the comment. Thanks for the catch!